### PR TITLE
Docs: Link New Coding Style

### PR DESCRIPTION
**Pull request by ax3l**
The new coding style was not yet linked. Updates it and adds a little rationale.

### CODE REVIEW

File: docs/source/dev/styleguide.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+   Our coding guide lines are documented in `this repository <https://github.com/ComputationalRadiationPhysics/contributing>`_.
+
+Source Style
+------------
+
+For contributions, *an ideal patch blends in the existing coding style around it* without being noticed as an addition when applied.
+Nevertheless, please make sure *new files* follow the styles linked above as strict as possible from the beginning.
+
+Unfortunately, we currently do not have tools available to auto-format all aspects of our style guidelines.
+Since we want to focus of the content of your contribution, we try to cover as much as possible by automated tests which you always have to pass.
```
Comment (PrometheusPi) [inline docs/source/dev/styleguide.rst]: REQUIRED:
typo `of` -> `on`

File: docs/source/dev/styleguide.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Please **add the according license header** snippet to your *new files*:
+
+* for PIConGPU (GPLv3+): ``src/tools/bin/addLicense <FileName>``
+* for libraries (LGPLv3+ & GPLv3+):
+  ``export PROJECT_NAME=libPMacc && src/tools/bin/addLicense <FileName>``
+* delete other headers: ``src/tools/bin/deleteHeadComment <FileName>``
+* add license to all ``.hpp`` files within a directory (recursive):
+  ``export PROJECT_NAME=PIConGPU && src/tools/bin/findAndDo <PATH> "*.hpp" src/tools/bin/addLicense``
+* the default project name ist ``PIConGPU`` (case sensitive!) and adds the GPLv3+ only
```
Comment (PrometheusPi) [inline docs/source/dev/styleguide.rst]: REQUIRED:
typo 1 `ist` -> `is`
typo 2 `adds` -> `add` since it is 2nd person singular

**Comment by ax3l** [comment]
>  Do you really want to delete the link to the coding style guide?

No, I added a link to the new coding guide line on top :)

**Comment by PrometheusPi** [comment]
@ax3l but not in `CONTRIBUTING.md` - or did I miss id (again)?

**Comment by ax3l** [comment]
> but not in CONTRIBUTING.md - or did I miss it (again)?

yes, I am in the process of ripping the file apart to the new docs and then just link there

**Comment by PrometheusPi** [comment]
Okay

**Pull request merged**